move dashboards file (#17476)
netdata/netdata: docs/glossary.md

## D

> - [**Dashboard**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allow you to make sense of your infrastructure and its health and performance.

Reviewer: ```suggestion
- [**Dashboard**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the-box visual representation of metrics to provide insight into your infrastructure, its health and performance.
```

I think that

> allow you to make sense of your infrastructure

sounds strange. To make sense of your infrastructure kek

> - [**Dashboards**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allows you to make sense of your infrastructure and its health and performance.

Reviewer: Should this be **Dashboard**? Why plural?

Author: yeah, I'll do it singular, nice catch

> - [**Dashboards**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allows you to make sense of your infrastructure and its health and performance.

Author: ```suggestion
- [**Dashboard**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allow you to make sense of your infrastructure and its health and performance.
```

Reviewer: > Out-of-the-box visual representation of metrics to provide insight into your infrastructure, its health and performance.
